shopelectro/views/catalog.py

```diff
@@ -71,9 +70,10 @@ def get_catalog_context(request, category, raw_tags, page_number, per_page, sort
     brands = newcontext.products.ProductBrands(paginated_products, all_tags)
     grouped_tags = newcontext.tags.GroupedTags(all_tags)
 
-    contexts = newcontext.Contexts(
-        all_tags, paginated_products, images, brands, grouped_tags,
-    )
+    contexts = newcontext.Contexts(paginated_products, images, brands, grouped_tags)
+    # @todo #683:60m Create context class(es).
```

Comment:
so, now we have two subtasks here:
`Create context class(es) for catalog page representation.`
and `Create context class(es).`

Seems like duplicate.
I propose to move details about `optional_context` to the first pdd subtask body.
And remove this task

Reply:
They are pretty different and separate. The first todo is about composition of classes for catalog views, and the second one will get rid the `optional_context` and patching of page instance

Reply:
I will clarify these points in tasks